### ISSUE TITLE
A few minor a11y and other issues

### ISSUE THREAD

**Issue by amellnik** [JuliaLang/www.julialang.org#548]
I noticed a few minor issues when looking at the draft website, some (all?) of which are probably common to the previous version.  I don't think any are blocking.  

- [ ] Text in top nav link has borderline contrast, and when the user hovers over the nav bar item the contrast drops to 3.91:1.  
- [ ] No change in visual style when the star portion of the github badge is tab-focused.  
- [ ] Contract is not great on the tab selectors (like under Ecosystem).  
- [ ] Revise logo is smooshed under essential tools.  
- [ ] Donate button at the top has almost invisible style change on focus. Donate button on the bottom is better. 
- [ ] Italic issue in NumFOCUS paragraph at `/jsoc`
- [ ] For most pages the nav item at the top gets a red underline when the user is on the page, but not for `/jsoc`
- [ ] The grey alert at the top about Prof Edelman's talk seems like it should have a button to hide it. 
- [ ] Dates for blog posts have low contrast.   
- [ ] Lots of images without `alt` set (I can help fill these in later).  

I haven't tried it with a screen reader yet but will give it a spin tomorrow.  Let me know if I should file this in the main website repo instead.

**Comment by abhishalya** [comment]
Thanks, this is very useful. But this doesn't seem to block https://github.com/JuliaLang/www.julialang.org/pull/544 so maybe this can be done post merge.

**Comment by logankilpatrick** [comment]
Yeah, that’s what Alex said too. We can tackle this after the merge.

**Comment by cormullion** [comment]
The [new and squarier Revise logo](https://github.com/cormullion/Revise.jl/blob/797048cebed9048134fd7bdacdc7736e3229b6a3/docs/src/assets/logo.png) is ready to go (https://github.com/timholy/Revise.jl/pull/413) but I thought it would be better to wait until the dust has settled a bit.

**Comment by ViralBShah** [comment]
@cormullion We can get the new revise logo in.

**Comment by cormullion** [comment]
Bump? (https://github.com/timholy/Revise.jl/pull/413#issuecomment-582077222)

**Comment by zdroid** [comment]
This should be updated.

1. This seems not to be an issue anymore:
> For most pages the nav item at the top gets a red underline when the user is on the page, but not for /jsoc
2. Blog post dates have regularcontrast... since they're just black text on white background?
3. Donate buttons are the same
4. GitHub's star badge does change on focus

Regarding 3. and 4, those are just buttons provided by GitHub. It's not really something made specifically for julialang.org.

**Comment by Keno** [comment]
It would be good to go through and do an alt text audit. That seems like it should be simple to do. Also, if we can have a CI give a warning on images that lack alt text that'd be great to keep up quality.
